pkgs/applications/networking/browsers/mullvad-browser/default.nix

```diff
@@ -235,7 +261,19 @@ stdenv.mkDerivation rec {
     runHook postBuild
   '';
 
-  installPhase = ''
+  installPhase = if stdenv.isDarwin then ''
```

Comment:
`lib.optionalString` would accomplish same you are doing. But I'm finding this pattern substitution weird here. I'm afraid this might disable install hooks and re-write `installPhase` to empty. But needs to be confirmed.